src/RF24LogLevel.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file RF24LogLevel.h
+ *
+ * Created on: 2 Oct 2020
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2020        Witold Markowski (wmarkow)
+ *    2021        Brendan Doherty (2bndy5)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+
+#ifndef SRC_RF24LOGLEVEL_H_
+#define SRC_RF24LOGLEVEL_H_
+
+#include <avr/pgmspace.h>
+
+/** the predefined logging levels */
+enum RF24LogLevel : uint8_t
+{
+   /** the level to disable all log messages */
+     OFF = 0b0000000,

```

Comment:
When you set a log level to OFF (which is 0) the idea is that all of standard log levels will be not logged. By this we disable logging (however it depends on the handler implementation).